remix.init/index.js

```diff
@@ -2,26 +2,80 @@ const { execSync } = require("child_process");
 const crypto = require("crypto");
 const fs = require("fs/promises");
 const path = require("path");
+
+const { toLogicalID } = require("@architect/utils");
 const inquirer = require("inquirer");
+const PackageJson = require("@npmcli/package-json");
 const YAML = require("yaml");
 
-const sort = require("sort-package-json");
-const { toLogicalID } = require("@architect/utils");
+const cleanupDeployWorkflow = (deployWorkflow, deployWorkflowPath) => {
+  delete deployWorkflow.jobs.typecheck;
+  deployWorkflow.jobs.deploy.needs = deployWorkflow.jobs.deploy.needs.filter(
+    (need) => need !== "typecheck"
+  );
 
-function getRandomString(length) {
-  return crypto.randomBytes(length).toString("hex");
-}
+  return [fs.writeFile(deployWorkflowPath, YAML.stringify(deployWorkflow))];
+};
+
+const cleanupVitestConfig = (vitestConfig, vitestConfigPath) => {
+  const newVitestConfig = vitestConfig.replace(
+    "setup-test-env.ts",
+    "setup-test-env.js"
+  );
+
+  return [fs.writeFile(vitestConfigPath, newVitestConfig)];
+};
+
+const getRandomString = (length) => crypto.randomBytes(length).toString("hex");
+
+const readFileIfNotTypeScript = (
+  isTypeScript,
+  filePath,
+  parseFunction = (result) => result
+) =>
+  (isTypeScript ? Promise.resolve() : fs.readFile(filePath, "utf-8")).then(
+    parseFunction
+  );
+
+const removeUnusedDependencies = (dependencies, unusedDependencies) =>
```

Comment:
This is a cool idea that could be used for the inverse if needed: adding new JS dependencies. 💡

Reply:
We prefer to remove the dependencies afterwards, as that makes it easier to update versions